src/components/pages/NoPageFound.js

```diff
@@ -0,0 +1,9 @@
+function NoPageFound() {
+  return (
+    <div className="currentPage">
```

Comment:
I recycled this component from one of my other unfinished projects. The "currentPage" class does not exist for this project.